### PR TITLE
ELECTRON-990 - add description to SSO checkbox in electron installer

### PR DESCRIPTION
**Pull request by vrbsm**
## Description
Add description to "SSO" check box in electron installer.
More information on [ELECTRON-990](https://perzoinc.atlassian.net/browse/ELECTRON-990)

## Solution Approach
Create a small label/ explanation in the installer: "only check this option if your Symphony POD has been configured for SSO, in doubt do not check - contract your Symphony Admin".

## Screenshot
- MacOS
electron-990

- Windows
electron-990-2
electron-990

## QA Checklist
- [x] Unit-Tests
- [ ] Automation-Tests

Attach unit & spectron tests results in PDF format against the above task lists for this branch

### CODE REVIEW

File: installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/Base.lproj/MyInstallerPane.xib [hunk 1]
```diff
@@ -191,6 +192,15 @@
                                             <action selector="handleSSOCheckboxStateChange:" target="Qsn-FY-4qK" id="LyC-VX-uCH"/>
                                         </connections>
                                     </button>
+                                    <textField hidden="YES" horizontalHuggingPriority="251" verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="xA2-1j-jeG">
+                                        <rect key="frame" x="15" y="246" width="340" height="27"/>
+                                        <autoresizingMask key="autoresizingMask"/>
+                                        <textFieldCell key="cell" alignment="left" title="Only check this option if your Symphony POD has been configured for SSO, in doubt do not check - Contract your Symphony Admin" id="qnN-I5-rxq">
```
Comment (KiranNiranjan) [inline installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/Base.lproj/MyInstallerPane.xib]:
```suggestion
                                        <textFieldCell key="cell" alignment="left" title="Only check this option if your Symphony POD has been configured for SSO, in doubt do not check - contact your Symphony Admin" id="qnN-I5-rxq">
```

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **Victor R. Braga de Sales Mascarenhas**.
This is most likely caused by a git client misconfiguration; please make sure to:
1. check if your git client is configured with an email to sign commits `git config --list | grep email`
2. If not, set it up using `git config --global user.email email@example.com`
3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

**Comment by VishwasShashidhar** [comment]
@vrbsm Please add screenshots directly here. We can't access Google Drive link.

File: installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/Base.lproj/MyInstallerPane.xib [hunk 2]
```diff
@@ -191,6 +192,15 @@
                                             <action selector="handleSSOCheckboxStateChange:" target="Qsn-FY-4qK" id="LyC-VX-uCH"/>
                                         </connections>
                                     </button>
+                                    <textField hidden="YES" horizontalHuggingPriority="251" verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="xA2-1j-jeG">
+                                        <rect key="frame" x="15" y="246" width="340" height="27"/>
+                                        <autoresizingMask key="autoresizingMask"/>
+                                        <textFieldCell key="cell" alignment="left" title="Only check this option if your Symphony POD has been configured for SSO, in doubt do not check - contract your Symphony Admin" id="qnN-I5-rxq">
```
Comment (VishwasShashidhar) [inline installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/Base.lproj/MyInstallerPane.xib]: typo here. `contract` should be `contact`

**Pull request merged**